internal/core/command/config/config.go

```diff
@@ -34,6 +35,29 @@ type WritableInfo struct {
 	InsecureSecrets bootstrapConfig.InsecureSecrets
 }
 
+type MessageQueue struct {
+	Internal InternalMessageQueue
+	External ExternalMQTT
+}
+
+type InternalMessageQueue struct {
+	bootstrapConfig.MessageBusInfo
+	RequestTopicPrefix                 string
+	ResponseTopic                      string
+	InternalRequestCommandTopic        string
+	InternalResponseCommandTopicPrefix string
+	InternalRequestQueryTopic          string
+	InternalResponseQueryTopic         string
+}
+
+type ExternalMQTT struct {
+	bootstrapConfig.MessageBusInfo
```

Comment:
Must be a new struct like here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/523a62ae167beff3074b878453cb29e7723133bf/internal/common/config.go#L125-L155
This eliminates the "optional" section since we know what is needed for MQTT.
If you copy this to bootstrap it can then be reused in App SDK later. 
Also would like for the External MQTT handler to be written in a way that it can be reused from the APP SDK.

GetBootstrap() below will also need to return this so the new handler can get at the needed config. i.e. addition in go-mod-
GetBootstrap() below will also need to return this so the new handler can get at the needed config. i.e. addition in go-mod-bootstrap.

Reply:
put ExternalMqttConfig struct to go-mod-bootstrap